packages/trip-details/src/index.tsx

```diff
@@ -72,6 +72,7 @@ export function TripDetails({
   fareDetailsLayout,
   fareKeyNameMap = {},
   itinerary,
+  showApproximatePrefixNonTransitLegs,
```

Comment:
Can you illustrate this new prop in Storybook, please (Create a new story or modify an existing one)?